stages/org.osbuild.users

```diff
@@ -69,7 +70,8 @@ def usermod(root, name, gid=None, groups=None, description=None, home=None, shel
         arguments += ["--expiredate", str(expiredate)]
 
     if arguments:
-        subprocess.run(["chroot", root, "usermod", *arguments, name], check=True)
+        with Chroot(root) as chroot:
```

Comment:
This test will stop working when run as non-root with this change (c.f. https://github.com/osbuild/osbuild/pull/1936#discussion_r1856176540)